docs/Project-Configuration/Project-Configuration-Files.md

```diff
@@ -32,6 +32,7 @@ For example, a couple of tasks might look like:
 ``` yaml
 tasks:
 - name: compile
+  exec_timeout_secs: 20
```

Comment:
Since this is meant to demonstrate that below "will override the project level exec_timeout defined above for this task", we should probably make this a different value than the one below.